src/test/java/hudson/plugins/git/UserMergeOptionsTest.java

```diff
@@ -108,13 +114,19 @@ public void testGetFastForwardMode() {
         assertEquals(expectedFastForwardMode == null ? MergeCommand.GitPluginFastForwardMode.FF : expectedFastForwardMode, options.getFastForwardMode());
     }
 
+    @Test
+    public void testTimeout() {
+        assertEquals(expectedTimeout, options.getTimeout());
```

Comment:
It is good to assert that the option was set correctly.  Thanks for doing that.

Unfortunately, that doesn't confirm that the timeout was set for the actual git command.  That needs to be checked with the logContains() assertion to confirm that the `# timeout=XXX` is on the merge statement that is logged.

Reply:
Will be writing a test soon for that :+1: